src/patternfly/components/Button/examples/button-states-example.hbs

```diff
@@ -1,49 +1,64 @@
 {{#> button button--modifier="pf-m-primary"}}
+  {{> button-icon button-icon--modifier="pf-c-button__icon"}}
   Primary
 {{/button}}
 {{#> button button--modifier="pf-m-primary pf-m-focus"}}
+  {{> button-icon button-icon--modifier="pf-c-button__icon"}}
   Primary Focus
 {{/button}}
 {{#> button button--modifier="pf-m-primary pf-m-active"}}
+  {{> button-icon button-icon--modifier="pf-c-button__icon"}}
   Primary Active
 {{/button}}
 {{#> button button--modifier="pf-m-primary" button--attribute="disabled"}}
+  {{> button-icon button-icon--modifier="pf-c-button__icon"}}
   Primary Disabled
 {{/button}}
 <br><br>
 {{#> button button--modifier="pf-m-secondary"}}
+  {{> button-icon button-icon--modifier="pf-c-button__icon"}}
```

Comment:
None of those modifier lines are necessary, are they?  We should only add button-icon when an icon is necessary.

Reply:
Alright, I'll have another look. Thanks!